### PR TITLE
test-controller-story-author

### PR DESCRIPTION
**Pull request by tunglam96hd**

### CODE REVIEW

**Comment by tunglam96hd** [comment]
https://edu-redmine.framgia.vn/issues/5466

**Comment by tunglam96hd** [comment]
image

**Comment by tunglam96hd** [comment]
ready

File: spec/controllers/authors_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    it "render show author" do
+      get :show, params: {id: author.id}
+      expect(response).to render_template "show"
+    end
+  end
+
+  describe "POST #create" do
+    context "with valid attributes" do
+      before {post :create,
+        params: {author: FactoryBot.attributes_for(:story, member_id: member.id)}}
```
Comment (huongnguyenmta) [inline spec/controllers/authors_controller_spec.rb]: author mà fake attribute story hở em (DEAD)

**Comment by tunglam96hd** [comment]
image

**Comment by tunglam96hd** [comment]
ready

**Comment by tunglam96hd** [comment]
ready

**Pull request merged**